pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -409,11 +409,11 @@ func (plugin *cniNetworkPlugin) buildCNIRuntimeConf(podName string, podNs string
 			// https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/README.md
 			// Rates are in bits per second, burst values are in bits.
 			bandwidthParam.IngressRate = int(ingress.Value())
-			bandwidthParam.IngressBurst = math.MaxInt32 // no limit
```

Comment:
Should we add something like... "in practice, limiting to 2Gbit is the equivalent of setting no limit`?

Reply:
Yeah, sounds reasonable.